README.md

```diff
@@ -75,3 +75,13 @@ Topic:topic2    PartitionCount:5        ReplicationFactor:1     Configs:retentio
 ```
 
 Any changes to the `test-topics.yml` file will be applied to the existing topics at each subsequent run.
+
+## Docker
+
+The Docker image is available from Docker Hub.
+
+An example on how to run the image assuming you know the address for zookeeper `<zookeeper_address>` and `config.yml` is in `<host_directory_with_config>`:
```

Comment:
I would change this to the following - I believe it reads more clearly.

>  Assuming you know the `<zookeeper_address>`, and have placed your config file named `config.yml` inside the `<config_dir_on_host>` directory on the Docker host, an example of how to run the image is:

Obviously you'll need to change the `host_directory_with_config` placeholder in the command below as well.